src/Tasks/Common/FileUtilities.MetadataReader.cs

```diff
@@ -13,6 +13,8 @@
 #if NETCOREAPP || !EXTENSIONS
 
 using System;
+using System.Collections.Generic;
+using System.Diagnostics;
```

Comment:
nit: Redundant namespace import. Debugging leftover?

Reply:
Yeah, I was trying to put an assert in but couldn't figure out how to phrase it.

Reply:
```suggestion
```